nasa/cFS-GroundSystem
# Fix #72 and #71, Upgrades to PyQt 5 and sequence count fix

**Describe the contribution**
This is a major change, upgrading PyQt4 to PyQt5 and includes a lot of cleanup/refactoring, and changes to the GUI itself

Fix #71 
Fix #72

**Testing performed**
Steps taken to test the contribution:
1. Make changes
2. Run software to see that it works

**Additional context**
PLEASE NOTE: All `.ui` files as well as any `.py` files auto-generated by `pyuic5` should be ignored during review, as these are related to the GUI itself

**Contributor Info - All information REQUIRED for consideration of pull request**
Leor Bleier, NASA GSFC\Code 582
